app/auth/authenticate.js

```diff
@@ -22,21 +22,44 @@ export async function getAuth (request, getJwtPublicKeyFunc = getJwtPublicKey) {
     if (!authHeader) {
       return {}
     }
-    logger.verbose('#DAL - Request authentication - Check verification', { code: DAL_REQUEST_AUTHENTICATION_001 })
+    logger.verbose('#DAL - Request authentication - Check verification', {
```

Comment:
we should really pass in the logger from app/graphql/context.js so it has requestID and remoteAddress already. Could add a seperate task for this tho

Reply:
in future good idea, but needs additional changes and considering limited time plus SOC testing I will leave this as is.